app/views/candidate_interface/course_choices/have_you_chosen.html.erb

```diff
@@ -0,0 +1,19 @@
+<% content_for :title, page_title('have_you_chosen') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @choice, url: candidate_interface_course_choices_choose_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <div class="govuk-!-margin-top-6">
+        <%= f.govuk_radio_buttons_fieldset :choice, legend: { size: 'xl', text: t('page_titles.have_you_chosen') } do %>
```

Comment:
Another smol thing:

Ideally there should be a larger margin between the heading and the radio buttons like the prototype. Couldn't seem to pass through the CSS class for the `govuk-!-margin-bottom-8` overwrite though.

Reply:
Hmm, I'm getting a little confused about the layout. I've added this as a separate checklist item to https://trello.com/c/HaxxM4vH/196-selecting-a-course-%F0%9F%8F%88-kickoff to make sure all the form pages look the same.